common/djangoapps/util/model_utils.py

```diff
@@ -0,0 +1,101 @@
+"""
+Utilities for django models.
+"""
+from eventtracking import tracker
+
+from django.core.exceptions import ObjectDoesNotExist
+from django.db.models.fields.related import RelatedField
+
+from django_countries.fields import Country
+
+
+def get_changed_fields_dict(instance, model_class):
```

Comment:
I would expect a unit test for this method, that checks for instance that related fields are not considered.